.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## PR Type
```

Comment:
PR Type is unnecessary IMO, you can describe the type by using prefix in the title, such as: fix, feat, doc, etc

Reply:
Please remove this section.